examples/getting-started-movielens/03-Training-with-TF.ipynb

```diff
@@ -2,7 +2,7 @@
  "cells": [
```

Comment:
<p>Can we add the tags to the previous notebook ( 02 NVTabular)?</p><p> </p><p>I see that we add another NVTabular workflow in this notebook - which just adds tags to the workflow. I think it will be cleaner, if we add the tags in the previous notebook and do not require two NVTAbular workflows.</p><p> </p><p>We can remove the gerne columns from the schema file to not load/create an archtecture with the feature</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/Merlin/pull/779/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Reply:
done